### PR TITLE
Add tooltip annotation for gold_path_field

### PR DESCRIPTION
**Pull request by aclum**
This PR adds a tool tip for slot gold_path_field. Description change only, no changes to example data. No migration needed.

### CODE REVIEW

**Comment by turbomam** [comment]
@aclum can you say where you got that definition?

You wanted me to do something similar for the MIxS environmental triad as a group, right?

**Review by mslarae13 — APPROVED** [review]

File: src/schema/nmdc.yaml [hunk 1]
```diff
@@ -2968,6 +2968,10 @@ slots:
     abstract: true
     description: >-
       This is a grouping for any of the gold path fields
+    annotations:
+      tool_tip:
```
Comment (turbomam) [inline src/schema/nmdc.yaml:2972]: single line

**Comment by aclum** [comment]
@turbomam  that description came from https://gold.jgi.doe.gov/ecosystem_classification, should i add that as comment?

**Comment by aclum** [comment]
@turbomam  okay to merge this in? This was tentatively approved by the refactor folks yesterday.

**Comment by turbomam** [comment]
I was hoping we could put the comment on the annotation itself. But that would require the use of a wordier [collection forms
](https://linkml.io/linkml-model/latest/docs/specification/06mapping/) and a nested annotation!

```yaml
    annotations:
      tool_tip:
        tag: tool_tip
        value: GOLD Ecosystem Classification paths describe the surroundings from which an environmental sample or an organism is collected.
        annotations:
          source: https://gold.jgi.doe.gov/ecosystem_classification
```

I will approve now. We can use either what I suggested above or what you already have.

**Review by turbomam — APPROVED** [review]

**Comment by aclum** [comment]
I don't have a preference, I just didn't have an example to work off of. I'll update to this format now.

**Comment by aclum** [comment]
@turbomam  updated the modeling to your preferred syntax.

File: src/schema/nmdc.yaml [hunk 2]
```diff
@@ -2968,6 +2968,12 @@ slots:
...
```
Comment (eecavanna) [inline src/schema/nmdc.yaml:2972]: I don't think I've come across this spelling of "tooltip" before. I don't meant to say it's incorrect, just that I think I've only ever seen it spelled as a single word.

**Pull request merged**